contribution-guide.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The Ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is licensed under the Apache license and is nurtured by all the contributors of the Ballerina community.
```

Comment:
Shouldn't we provide the link for Apache license[1] here? 

[1] https://www.apache.org/licenses/LICENSE-2.0

Reply:
Added the link.